### PR TITLE
Refine Manila installation instructions (SCRD-8734)

### PR DESCRIPTION
**Pull request by cadenzajon**

### CODE REVIEW

File: xml/installation-ardana-manila.xml [hunk 1]
```diff
@@ -70,38 +71,41 @@
...
+   </para>
+  </note>
  </section>
  <section>
   <title>Configure &o_sharefs; backend</title>
   <procedure>
    <step>
     <para>
-     Add backend to manila.conf.j2
+     Configure a backend to in the &o_sharefs; configuration file, following
```
Comment (bryanstephenson) [inline xml/installation-ardana-manila.xml]: Either change "to" -> "too" or perhaps just say "Configure a backend in the ..."

File: xml/installation-ardana-manila.xml [hunk 2]
```diff
@@ -21,7 +21,8 @@
    manila-data, manila-scheduler, manila-share, and messaging queue.
   </para>
   <para>
-   The following steps will install &o_sharefs;:
+   The following steps will install &o_sharefs; if it is not already present
```
Comment (asettle) [inline xml/installation-ardana-manila.xml]: Remove "will".

File: xml/installation-ardana-manila.xml [hunk 3]
```diff
@@ -70,38 +71,41 @@
...
+&prompt.ardana;manila service-list</screen>
     <para>
      The &o_sharefs; CLI can be run from &clm; or controller nodes.
     </para>
    </step>
   </procedure>
+  <note>
+   <para>
+    The <literal>manila-share</literal> service component is not started by the
+    <filename>manila-deploy.yml</filename> playbook under all conditions. This
```
Comment (asettle) [inline xml/installation-ardana-manila.xml]: "... is not started when the <filename>manila-deploy.yml</filename> playbook is run under all conditions." (just helps clarify the context)

File: xml/installation-ardana-manila.xml [hunk 4]
```diff
@@ -70,38 +71,41 @@
...
+    component requires that a valid backend be configured, which will be
```
Comment (asettle) [inline xml/installation-ardana-manila.xml]: which is described *

File: xml/installation-ardana-manila.xml (hunk 1 above)
Comment (asettle) [reply xml/installation-ardana-manila.xml]: Or:
"Configure the backend in the &o_sharefs; configuration file,..."

**Comment by asettle** [comment]
Thanks @cadenzajon !

**Pull request merged**